src/popup/popup.js

```
@@ -125,14 +316,15 @@
     logDebug("item", item);
     logDebug("suggest", suggest);
     suggest({
-        filename: 'bigmachines/' + 
-                 sanitizeFilename(bmSiteSubDomain) + '/' + 
-                 sanitizeFilename(bmSiteType) + '/' + 
-                 sanitizeFilename(item.filename),
+        filename: 'bigmachines/' +
+            sanitizeFilename(bmSiteSubDomain) + '/' +
+            sanitizeFilename(bmSiteType) + '/' +
+            (bmRuleType ? sanitizeFilename(bmRuleType) + '/' : '') +
+            sanitizeFilename(item.filename),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)